docs/getting_started/tutorial.md

```diff
@@ -507,32 +540,34 @@ Modify your `BlogIndexPage` model like this:
 class BlogIndexPage(Page):
     intro = RichTextField(blank=True)
 
+    # add the get_context method:
+
     def get_context(self, request):
         # Update context to include only published posts, ordered by reverse-chron
         context = super().get_context(request)
         blogpages = self.get_children().live().order_by('-first_published_at')
         context['blogpages'] = blogpages
         return context
+
+    # ...
 ```
 
 Here is a quick breakdown of the changes we made:
 
-1.  Retrieve the original context
-2.  Create a custom QuerySet modifier
-3.  Add it to the retrieved context
-4.  Return the modified context back to the view
+1. Retrieve the original context.
+2. Create a custom QuerySet modifier.
+3. Add it to the retrieved context.
+4. Return the modified context back to the view.
```

Comment:
These indents are added intentionally by the default Prettier markdown formatting, I know it's odd but its meant to be that way for larger numbers to have a consistent visual indent in the code.